.pre-commit-config.yaml

```
@@ -29,6 +31,15 @@ repos:
         name: run identity check
       - id: check-hooks-apply
         name: run check hooks apply
+  - repo: local
+    hooks:
+      - id: prettier
+        name: run prettier
+        description: format files with prettier
+        entry: prettier --write '**/*.yaml' '**/*.yml'
+        files: \.ya?ml$
+        language: node
+        additional_dependencies: ['prettier@3.3.3']
```

```suggestion
        additional_dependencies: ['prettier@3.5.3']
```

https://www.npmjs.com/package/prettier